schematic/models/metadata.py

```diff
@@ -39,6 +40,7 @@ def __init__(
         inputMModelLocation: str,
         inputMModelLocationType: str,
         data_model_labels: str,
+        data_model_graph_pickle: Optional[str] = None,
```

Comment:
Why does this method only take the pickle file, where the one above takes the pickle file and a graph object?

Reply:
My impression is that this function would read from a file, not a graph object.